src/shared/i18n/I18NFrench.cpp

```diff
@@ -18,441 +18,441 @@
 #include "I18N.hpp"
 
 const I18N::Text I18N::all_texts_french[] = {
-    { I18N_BUTTON_YES, "Yes" },
-    { I18N_BUTTON_NO, "No" },
+    { I18N_BUTTON_YES, "Oui" },
+    { I18N_BUTTON_NO, "Non" },
     { I18N_BUTTON_OK, "Ok" },
-    { I18N_BUTTON_CANCEL, "Cancel" },
-    { I18N_BUTTON_CLOSE, "Close" },
-    { I18N_BUTTON_START_SERVER, "Start Server" },
-    { I18N_BUTTON_DELETE, "Delete" },
+    { I18N_BUTTON_CANCEL, "Annuler" },
+    { I18N_BUTTON_CLOSE, "Fermer" },
+    { I18N_BUTTON_START_SERVER, "Démarrer le Serveur" },
+    { I18N_BUTTON_DELETE, "Supprimer" },
 
-    { I18N_TILETYPE01, "non blocking" },
-    { I18N_TILETYPE02, "blocking" },
-    { I18N_TILETYPE03, "falling only blocking" },
-    { I18N_TILETYPE04, "red base" },
-    { I18N_TILETYPE05, "blue base" },
-    { I18N_TILETYPE06, "killing" },
+    { I18N_TILETYPE01, "non bloquant" },
+    { I18N_TILETYPE02, "bloquant" },
+    { I18N_TILETYPE03, "bloquant juste la tombée" },
+    { I18N_TILETYPE04, "base rouge" },
+    { I18N_TILETYPE05, "base bleue" },
+    { I18N_TILETYPE06, "mortel" },
     { I18N_TILETYPE07, "jump, no descend" },
```

Comment:
"saut, pas de descente". Not really meaningful per se, would have to check in the game's context (I guess it's for the editor).